migrations/2021-08-23-161500_aggregated_ft_nft_tables/up.sql

```diff
@@ -0,0 +1,62 @@
+CREATE TABLE aggregated__fungible_tokens
+(
+    id                                bigserial                NOT NULL,
+    included_in_transaction_hash      text                     NOT NULL,
+    included_in_transaction_timestamp numeric(20, 0)           NOT NULL,
+    transaction_status                execution_outcome_status NOT NULL,
```

Comment:
Will we actually set the **transaction** status? Can we actually say if the FT operation was successful as a whole transaction rather than just a single receipt execution outcome status? (I have yet to read the FT standard, so keep that in mind) How would that play with cross-contract calls (e.g. FT operation is triggered as part of some other transaction just as a cross-contract call)